_posts/2015-04-30-gsoc-15-is-now-live.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Google Summer of Code 2015 is now live
+categories:
+- activism
+author: febuiles
+published: true
+date: 2015-04-30 15:00:00 +00:00
+---
+**TODO** Confirm Josh K. participation and remove **TODO** tags.
+
+Google has announced the list of [accepted projects][melange] for this year's [Google Summer of
+Code][gsoc] (GSoC). Rails has been granted 8 slots, here's a brief introduction to the projects and
+the people behind them.
+
+## Performance Enhancements for the Asset Pipeline
+
+**Student:** [Martha de Luque]()    
+**Mentors:** [Guillermo Iguaran](https://github.com/guilleiguaran) and [Joshua Peek](https://github.com/josh)
```

Comment:
Woops, sorry, I went through the Basecamp list instead of the actual Melange applications. I’ve pushed a fix.

> On Apr 30, 2015, at 6:39 PM, Godfrey Chan <notifications@github.com> wrote:
> 
> In _posts/2015-04-30-gsoc-15-is-now-live.markdown:
> 
> > +categories:
> > +- activism
> > +author: febuiles
> > +published: true
> > +date: 2015-04-30 15:00:00 +00:00
> > +---
> > +**TODO** Confirm Josh K. participation and remove **TODO** tags.
> > +
> > +Google has announced the list of [accepted projects][melange] for this year's [Google Summer of
> > +Code][gsoc] (GSoC). Rails has been granted 8 slots, here's a brief introduction to the projects and
> > +the people behind them.
> > +
> > +## Performance Enhancements for the Asset Pipeline
> > +
> > +**Student:** [Martha de Luque]()    
> > +**Mentors:** [Guillermo Iguaran](https://github.com/guilleiguaran) and [Joshua Peek](https://github.com/josh)
> 
> This should be Guillermo and @rafaelfranca
> 
> —
> Reply to this email directly or view it on GitHub.
>